test/conformance/conformance_parallel_reduce.cpp

```diff
@@ -56,6 +57,67 @@ struct ReduceBody {
     }
 };
 
+// The type that is copyable and can be stored in the container
+// that would be copied only in the empty state.
+template <typename T>
+class NeverCopyWrapper {
+public:
+    NeverCopyWrapper() = default;
+    NeverCopyWrapper(const T& obj) : my_obj(obj) {}
+
+    NeverCopyWrapper(NeverCopyWrapper&&) = default;
+    NeverCopyWrapper& operator=(NeverCopyWrapper&&) = default;
+
+    NeverCopyWrapper(const NeverCopyWrapper&) {
+        REQUIRE_MESSAGE(false, "Copy constructor of NeverCopyWrapper should never be called");
+    }
+
+    NeverCopyWrapper& operator=(const NeverCopyWrapper&) {
+        REQUIRE_MESSAGE(false, "Copy assignment of NeverCopyWrapper should never be called");
+        return *this;
+    }
```

Comment:
Wouldn't it be better to `=delete;` these?

Reply:
I have investigated one more time, and since we don't use `std::vector` directly, only using the wrapper, we can control that actual copy constructor would not be called by the container. So I removed `NeverCopyWrapper` and changed it to be `MoveOnlyWrapper`